config/credentials.yml.enc

```diff
@@ -1 +1 @@
-3A2iHaX4YzSo+IWdkr55uY6UiMSObrJaldkEvMFVYjKwf2i4XcHq1eehjd/1gozeAWFBiIwxuk2sZw51oizpxhH/4W6FMtxxGxRw9omwRnnCH9h0eRv/FBlNI5q9AbOMEw+p1vlimWi8L3OvY8Jvz6Z6FllxJOmRa1ArYYFaQYxZ0rzslBGwCUkywAGwt4nStKZEWdK63ORPGtAj79oSa/7pZnRXFkY+s95+7wCk1RfSnkoYDB6XB16lxBUJ1rOmwlVhDdvJRYNBTPMf2HpqSBiDIvDnUhe6RNbQ0E4ZYoSMOaY++5hZnSoU7BC854+/5NW4qzmYqhCYoeUunookk8eJ4cFr4gJAx3ZODb6tg0FeEEqD2z3WmBwdZzbUXp2hJb/4OVYTjQI53RcJarV8YfIrIZmWPyA23CBY--LfRMgKksXhzOk/9l--FbLnIqIjsysVwKi5GfxJvA==
\ No newline at end of file
+SyED25dz8KIGfkq+GEUsIPNCxm91b5Xq8ejdqoqzH7U0Q/poHpD6slnL5SpEwIAcEdrqvhx9H2HN2uX9dZLw43JsVtRNin8ac73H3cklE7p7v03glJY3gRKpcOwQrl+gk2QktWe4WV/68kCNKEswA0f4gGx/p5YO6dPEHLVxTNgk2BXl+AOdwmL/kY7ZwJXfB5YRqBt37b3r1PQqOVsKKtwezeVwNidAo2pFZqHyzAVIAn570evO9gEcBFQXIIrYt9sOS77ajXz8KAGueBW7qVFwWYGSeIo4A3ERdZw/AbEFVvPunT5xM7ni9Vhv677wbQ3tcH0HDgiWzQLfj7fQuPd5Sh6csEm9EBgfXYXLj+3hhMDQvqIO78E1ZXE26M3KBkLppnwo4B0dVbGfNdATdJ34itXT3ktXm+Ooh2X0NzGZOamNH4fZEUqO9OhZCp95vp4Or1xryqmWW+ScS2PQ12AbqqswExVClp7rOLbcybY=--aHMk8TzvKLCHUpmz--K9IgN9csxkwgfL7CGkDSgw==
```

Comment:
please add a new line at the end of the file

Reply:
I tried to add a new line at the end of this file, but the system is not permitting me to save it. first, I edited this file because I believed that it needed to contain the API token from Postmark to function properly. However, I removed it once I realized that everything operates effectively through  the letter_opener gem.

Paths to Solution Attempted So Far:
I edited the file in VS Code, but this approach failed. Editing is not permitted because the file is encrypted.
I edited the file using `EDITOR="nano" rails credentials:edit`, but the new line does not appear in VS Code.
I edited the file in `EDITOR="code --wait" rails credentials:edit` mode, but the new line still doesn’t appear.